wrappers/python/examples/wnt/README.md

```diff
@@ -0,0 +1,64 @@
+# WNT API Examples
+
+This folder contains the WNT API Examples.
+
+## Installation
+
+The examples require that the wirepas_messaging packet is installed. Please see the README.md
+file in /wrappers/python folder for installation instructions.
+
+## WNT backend settings
+
+For running the examples, WNT backend connection parameters needs be given by creating settings.yml
+file to the same folder from where the examples are run. For example settings file please see
+settings.yml in the root folder.
```

Comment:
It is not clear to me what is the root folder you talk about here. The repository or the examples root folder? It would also be good to provide an inline example of the settings for easy copy paste.

Reply:
Done